website/addons/dataverse/static/dataverseFangornConfig.js

```diff
@@ -29,29 +29,55 @@ var _dataverseItemButtons = {
             tb.dropzone.hiddenFileInput.click();
             tb.dropzoneItemCache = item;
         }
-        function dataversePublish(event, item, col) {
+        // Get the external Dataverse host before publishing
+        function getDataverseHost(event, item, col) {
+            var host;
+            $.ajax({
```

Comment:
@asanfilippo7, the info in `item.data` is coming from a view function somewhere in the dataverse addon code (probably `dataverse/views.py`, so even though the host info is not currently in `item.data`, it's possible to add it in by updating what's returned by that view function. That way, you don't have to make an additional request to get that info.